frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -439,10 +439,15 @@ class BitBox02 extends Component<Props, State> {
                     <div className="flex flex-1 scrollableContainer">
                         <Steps>
                             { (status === 'connected') ? (
-                                <Fullscreen withBottomBar>
-                                    <FullscreenHeader title={t('button.unlock')} />
-                                    <FullscreenContent>
+                                <Fullscreen withBottomBar width="600px">
```

Comment:
How does it work on device screens smaller than 600px, does it somehow cancel out?

Reply:
This will be applied as inline style `style="width: 600px"` and on the same element there is `max-width: 100%;`, so it should never be larger that 100% width.

Reply:
ah, `max-width` explains it :+1: